tests/test_client_functional.py

```diff
@@ -2333,25 +2335,74 @@ async def test_creds_in_auth_and_url() -> None:
         await session.close()
 
 
-async def test_drop_auth_on_redirect_to_other_host(aiohttp_server: Any) -> None:
+@pytest.mark.parametrize(
+    ["url_a", "url_b", "drop_header"],
```

Comment:
It's better to use meaningful variable names. What are "a" and "b"? `url_from`/`url_to`?

Reply:
s/drop_header/is_drop_header_expected/

Reply:
Fair enough! Done.